src/main/java/seedu/address/ui/CalendarCard.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.ui;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.Region;
+import seedu.address.model.session.Session;
+import seedu.address.model.tuition.Tuition;
+
+public class CalendarCard extends UiPart<Region> {
+    private static final String FXML = "CalendarCard.fxml";
+    private Tuition tuition;
+
+    @FXML private Label periodFxml;
+    @FXML private Label nameFxml;
+    @FXML private Label subjectFxml;
```

Comment:
Maybe can add a javadoc comment for a description of the CalendarCard class?

Reply:
got it!